build.gradle

```diff
@@ -182,16 +182,18 @@ repositories {
 def versions = [
   bouncycastle: '1.61',
   flywayV: plugins.getPlugin('org.flywaydb.flyway').class.package.implementationVersion,
-  junit: '5.4.2',
-  mockitoJupiter: '2.27.0',
+  junit: '5.5.0',
+  junitPlatform: '1.5.0',
+  mockitoJupiter: '3.0.0',
   pdfbox: '2.0.15',
   reformLogging: '5.0.1',
-  restAssured: '3.3.0',
   shedlock: '2.5.0',
   springBoot: plugins.getPlugin('org.springframework.boot').class.package.implementationVersion,
   springfoxSwagger: '2.9.2'
 ]
 
+ext["rest-assured.version"] = '4.0.0'
```

Comment:
why exactly does it have to be done this way?

Reply:
assures all dependencies are imported with the same version. the dependencies are:

- `io.rest-assured:json-path`
- `io.rest-assured:rest-assured-common`
- `io.rest-assured:xml-path`

otherwise **only** `io.rest-assured:rest-assured` will be pulled in as version 4, those ^ - version 3